ansible-scylla-node/tasks/adjust_keyspace_replication.yml

```diff
@@ -24,9 +24,23 @@
   loop: "{{ groups['scylla'] }}"
   run_once: true
 
+- set_fact:
+    scylla_admin_username: "{{ scylla_admin_default_user }}"
+    scylla_admin_password: "{{ lookup('ini', scylla_admin_default_user, section='cql_credentials', allow_no_value=true, default=omit, file=inventory_file) }}"
+  when: "scylla_admin_default_user is defined and 'cql_credentials' in groups"
```

Comment:
Why do you put the "when" expression in quotes?

Reply:
quotes removed